.gitignore

```diff
@@ -3,4 +3,6 @@
 /node_modules
 /bower_components
 /release.properties
-/pom.xml.releaseBackup
\ No newline at end of file
+/pom.xml.releaseBackup
+*.iml
+.idea
```

Comment:
Can we not add IDE specific .gitignore? In an open source project this can become a mess because each contributor uses a different environment.

Reply:
I would add them exactly because every contributor uses a different environment. In my view, no IDE-specific files should be in the repo at all (unless there is a good reason for it). That's why I would also remove the Eclipse-files and ignore them. This allows everybody to configure the project the way they like and should also not break anything.

Reply:
I agree that no IDE-specific files should be in the repo. In fact, I would go one step further and say that no IDE-specific `.gitignore` entries should be in the repo and instead contributors (including myself), should ignore them with a [global gitignore](http://egorsmirnov.me/2015/05/04/global-gitignore-file.html).

Do you reckon it makes sense to have contributors to have their own IDE-specific files gitignored globally?

Reply:
From experience, contributors do not have this setup in the global gitignore. That's why I would add the most common IDEs to the .gitignore, to make contributing as easy as possible without accidentally committing irrelevant files.

Reply:
That alright then